documentation/project-docs/repro-helix-failure.md

```diff
@@ -18,7 +18,7 @@ REM use build env to have stage 0 dotnet on the PATH
 .\artifacts\sdk-build-env.bat
 
 REM run the special test target CreateLocalHelixTestLayout. To have the test layout created on disk.
-dotnet msbuild /restore /t:CreateLocalHelixTestLayout .\test\UnitTests.proj /p:creator=dotnetsdkdev  /p:_CustomHelixTargetQueue=Windows.Server.Amd64.VS2019.Pre.Open /bl
+dotnet msbuild /restore /t:CreateLocalHelixTestLayout .\test\UnitTests.proj /p:creator=dotnetsdkdev  /p:CustomHelixTargetQueue=Windows.Server.Amd64.VS2019.Pre.Open /bl
```

Comment:
Renamed `_CustomHelixTargetQueue` to `CustomHelixTargetQueue` since the underscore usually indicates locality to a particular file. Here, this value is provided externally to the file it is used in.